### ISSUE TITLE
`gradle clean` doesn't work

### ISSUE THREAD

**Issue by boris-petrov** [gretty-gradle-plugin/gretty#223]
Running `gradle clean` blows up with:

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':integrationTests:buildSrc:gretty-integrationTest:compileGroovy'.
> Could not resolve all files for configuration ':integrationTests:buildSrc:gretty-integrationTest:compileClasspath'.
   > Could not find org.gretty:gretty:3.0.5-SNAPSHOT.
     Searched in the following locations:
       - file:/home/boris/code/gretty/build/privateRepo/org/gretty/gretty/3.0.5-SNAPSHOT/maven-metadata.xml
       - file:/home/boris/code/gretty/build/privateRepo/org/gretty/gretty/3.0.5-SNAPSHOT/gretty-3.0.5-SNAPSHOT.pom
       - https://repo.maven.apache.org/maven2/org/gretty/gretty/3.0.5-SNAPSHOT/maven-metadata.xml
       - https://repo.maven.apache.org/maven2/org/gretty/gretty/3.0.5-SNAPSHOT/gretty-3.0.5-SNAPSHOT.pom
     Required by:
         project :integrationTests:buildSrc:gretty-integrationTest
```

I believe it's because of [this](https://github.com/gretty-gradle-plugin/gretty/blob/gretty-3.x/integrationTests/buildSrc/gretty-integrationTest/build.gradle#L9) line.

Unfortunately, not being able to do that leads to problems like [this](https://github.com/gretty-gradle-plugin/gretty/pull/210#issuecomment-826131978).

@f4lco - any ideas how to fix this?

**Comment by f4lco** [comment]
I did not debug yet, but it makes sense: if the project is entirely cleaned, and Gradle tries to configure integration tests before building Gretty and publishing it to the private repository, then Gradle is bound to fail because of a missing dependency.

I have a reproduction by running "clean" twice in my working tree: the first invocation cleans out previously installed Gretty from the private repository, and the second invocation fails due to the "missing" dependency.

As a result, I think that the process with the private repository is dated, and would assume that implementing #199 would fix that. The proposal drops the private repository. Then instruct Gradle to built Gretty from the working tree, and to use the resulting artifacts as dependency in itests in one go.

**Comment by boris-petrov** [comment]
This has been fixed by #228.
